controller_interface/include/controller_interface/controller_interface.hpp

```diff
@@ -79,7 +79,8 @@ class ControllerInterface : public rclcpp_lifecycle::node_interfaces::LifecycleN
   void release_interfaces();
 
   CONTROLLER_INTERFACE_PUBLIC
-  virtual return_type init(const std::string & controller_name);
+  virtual return_type init(
+    const std::string & controller_name, const std::string & namespace_ = "");
```

Comment:
```suggestion
    const std::string & controller_name, const std::string & namespace = "");
```
I'm happy to put in a new PR fixing this naming convention in controller_manager too

Reply:
I tried this and unfortunately it doesn't work because `namespace` is reserved keyword in C++. 

Maybe naming it `ros_namespace` or `node_namespace` would be a solution for this ugly approach.

Reply:
Added in #695